src/lib/Zikula/Composer/ManuallyInstallAssets.php

```diff
@@ -29,6 +29,12 @@ class ManuallyInstallAssets extends ScriptHandler
     private static $assets = [
         '/jQuery.mmenu/dist/js/jquery.mmenu.all.min.js' => '/jquery-mmenu/js/jquery.mmenu.all.min.js',
         '/jQuery.mmenu/dist/css/jquery.mmenu.all.css' => '/jquery-mmenu/css/jquery.mmenu.all.css',
+        '/twitter/typeahead.js/dist/bloodhound.js' => '/typeahead.js/bloodhound.js',
```

Comment:
I would prefer `'/typeahead/bloodhound.js'` as a namespace in `/web`

also - you must adjust the paths in the code.